Susi/Controllers/DevicesActivityController/DeviceActivityMethods.swift

```diff
@@ -147,15 +142,16 @@ extension DevicesActivityViewController {
     }
 
     func setConfiguration() {
-        self.activityIndicator.startAnimating()
-        UIApplication.shared.beginIgnoringInteractionEvents()
+        self.passwordAlertController.dismiss(animated: true, completion: nil)
+        self.loadAlertIndicator(with: "Configuring..")
 
         Client.sharedInstance.setConfiguration(stt: "google", tts: "google", hotword: "y", wake: "n") { (success, message) in
             DispatchQueue.main.async {
                 if success {
+                    self.alertController.dismiss(animated: true, completion: nil)
                     print("Success!")
                 } else {
-                    self.activityIndicator.stopAnimating()
+                    self.alertController.dismiss(animated: true, completion: nil)
                     self.view.makeToast("", point: self.view.center, title: message, image: nil, completion: { didTap in
```

Comment:
Use of this?

Reply:
It is dismissing the `alertController` I am using as an indicator.